sleap/gui/app.py

```diff
@@ -1268,13 +1270,24 @@ def _has_topic(topic_list):
             self.update_cameras_model()
             self.update_unlinked_videos_model()
             self._update_sessions_menu()
+            self.update_instance_group_model()
 
     def update_unlinked_videos_model(self):
         """Update the unlinked videos model with the selected session."""
         self.sessions_dock.unlinked_videos_table.model().items = (
             self.labels._cache._linkage_of_videos["unlinked"]
         )
 
+    def update_instance_group_model(self):
+        session = self.state["session"]
+        if session is not None:
+            frame_idx: int = self.state["frame_idx"]
+            frame_group = session.frame_groups.get(frame_idx, None)
+            if frame_group is not None:
+                self.instance_groups_dock.table.model().items = (
+                    frame_group.instance_groups
+                )
```

Comment:
Should have an else statement for when we are on a `Video` without a `RecordingSession`